test/functional/setup.js

```diff
@@ -1,143 +1,149 @@
-var browserNatives     = require('testcafe-browser-natives');

```

Comment:
Strange, looks like that `git` have decided that file was rewritten. I'll reset the commit and then apply it back.